Makefile

```diff
@@ -45,17 +45,16 @@ help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
 ##@ Development
-lint-autofix: ## Autofix all possible linting errors.
-	golangci-lint run -E goimports --fix
+lint-autofix: golangci-lint ## Autofix all possible linting errors.
+	${GOLANGCI-LINT} run -E goimports --fix
 
 lint-manifests:
 	hack/lint-manifests.sh
 
-lint: lint-manifests
+lint: golangci-lint lint-manifests
 	go version
-	golangci-lint version
-	GO111MODULE=on golangci-lint run
-
+	${GOLANGCI-LINT} version
+	GO111MODULE=on ${GOLANGCI-LINT} run -c ./.golangci.yaml
```

Comment:
Is it needed? I thought, `.golangic.yml` would be taken by default

Reply:
Will remove it